pkgs/development/python-modules/ibmquantumexperience/default.nix

```diff
@@ -1,33 +0,0 @@
-{ lib
-, buildPythonPackage
-, fetchPypi
-, requests
-, requests_ntlm
-}:
-
-buildPythonPackage rec {
-  pname = "IBMQuantumExperience";
```

Comment:
please remove associated entry in pkgs/top-level/python-modules.nix

```
getting status of '/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/packet-spot-eval-1/pkgs/development/python-modules/ibmquantumexperience': No such file or directory
```

Reply:
ah, whoops. thanks.